### PR TITLE
Update financial minister voting requirements

### PR DESCRIPTION
**Pull request by TRManderson**
When we don't even have a financial minister defined, it'd be good to not require them to at the very least propose changes

### CODE REVIEW

**Comment by Guthers** [comment]
I think it may be more prudent for a person to change a matter relating to a section that has a designated  minister, then the minister is required to support it unless there is a majority from the others

**Comment by TRManderson** [comment]
My concern regarding the current state of things is that a minister can entirely block all legislation. With this mechanism, it becomes very very clear when the only reason that legislation has not been ratified is due to a blocking minister.

File: Constitution.md [hunk 1]
```diff
@@ -26,14 +26,14 @@
 
 1.1.2.a The constitution and all other documents may be edited or created with a majority vote.
 
-1.1.2.b Any member of the ruling party may propose a pull request for the change or creation of any document.
-
-1.1.2.b.a Only the financial minister may create pull requests affecting the Currency Act of 2016.
+1.1.2.b Any member of the ruling party may propose a pull request for the change or creation of any document, unless there is an exception under 1.1.2.b
```
Comment (CameronAavik) [inline Constitution.md:29]: I am not sure what you mean by

> exception under 1.1.2.b  

What kind of exception do you mean?

**Comment by lsenjov** [comment]
No votes, not ratified.

**Pull request closed without merging**